cmd/conf/homi/homi.yaml

```diff
@@ -0,0 +1,92 @@
+
+# generation type : docker, local, remote, deploy
+gen-type: docker
+
+# The output path for files (keys, genesis.json. conf, etc...)
+output: "homi-output"
+
+######################################################################
+#               For each node, test, validator keys                  #
+######################################################################
+
+# Network topology
+cn-num: 1
+pn-num: 0
+en-num: 0
+scn-num: 0
+spn-num: 0
+sen-num: 0
+test-num: 0
+validators-num: 0
+
+######################################################################
+#                       For deploy environment                       #
+######################################################################
+
+# Only for docker gen-type
+docker-image-id: klaytn/klaytn:latest
+subgroup-size: 21
```

Comment:
We can remove this line. `subGroupSizeFlag` is not used.

Reply:
Thanks!